admin/database/postgres/postgres.go

```diff
@@ -961,10 +961,10 @@ func (c *connection) FindUserAuthTokens(ctx context.Context, userID, afterID str
 	`)
 	args := []any{userID}
 	if afterID != "" {
-		qry.WriteString(" AND t.id > $2 ORDER BY t.id LIMIT $3")
+		qry.WriteString(" AND t.id > $2 ORDER BY t.created_on DESC, t.id DESC LIMIT $3")
```

Comment:
Won't this skip rows? The filter does not match the order by.

E.g. if you have `ORDER BY created_on DESC, id DESC`, your filter needs to be `created_on > ? OR (created_on = ? AND id > ?)`.

On reflection, timestamps are a very annoying pagination key. I suggest keeping ID pagination on the backend. Then in the CLI, use a large page size (e.g. 1000) and IF the result size is smaller than the page size, sort by timestamp in memory there. 

That will use creation time sorting for small lists, and switch to ID based pagination for huge lists. Should be good in practice.